### PR TITLE
[Notifier] Add documentation for telegram options

### LINKED ISSUE #14044: [Notifier] added telegram options
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#36496
| PR author(s) | @krasilnikovm
| Merged in    | 5.2-dev

### PR DESCRIPTION
**Pull request by CyberJack**
Add documentation about Telegram options (issue #14044)

### CODE REVIEW

File: notifier/chatters.rst [hunk 1]
```diff
@@ -152,3 +152,37 @@ some interactive options called `Embed elements`_::
...
+
+Adding Interactions to a Telegram Message
+-----------------------------------------
+
+With a Telegram message, you can use the
+:class:`Symfony\\Component\\Notifier\\Bridge\\Telegram\\TelegramOptions` to add
+`message options`_::
+
+    use Symfony\Component\Notifier\Bridge\Telegram\Reply\Markup\Button\KeyboardButton;
+    use Symfony\Component\Notifier\Bridge\Telegram\Reply\Markup\ReplyKeyboardMarkup;
```
Comment (wouterj) [inline notifier/chatters.rst]: These 2 use statements seems not to relate to the used classes (`InlineKeyboardButton` & `InlineKeyboardMarkup` are used instead). Can you please check which classname is correct and fix the other one? Thanks!

Comment (CyberJack) [reply notifier/chatters.rst]: The ones used in the example are correct. I updated the `use` statements in the documentation.

File: notifier/chatters.rst [hunk 2]
```diff
@@ -152,3 +152,37 @@ some interactive options called `Embed elements`_::
     $chatter->send($chatMessage);
 
 .. _`Embed elements`: https://discord.com/developers/docs/resources/webhook
+
+Adding Interactions to a Telegram Message
+-----------------------------------------
+
+With a Telegram message, you can use the
+:class:`Symfony\\Component\\Notifier\\Bridge\\Telegram\\TelegramOptions` to add
+`message options`_::
```
Comment (OskarStark) [inline notifier/chatters.rst]:
```suggestion
:class:`Symfony\\Component\\Notifier\\Bridge\\Telegram\\TelegramOptions` class
to add `message options`_::
```

File: notifier/chatters.rst [hunk 3]
```diff
@@ -152,3 +152,37 @@ some interactive options called `Embed elements`_::
     $chatter->send($chatMessage);
 
 .. _`Embed elements`: https://discord.com/developers/docs/resources/webhook
+
```
Comment (OskarStark) [inline notifier/chatters.rst]:
```suggestion
```

File: notifier/chatters.rst [hunk 4]
```diff
@@ -152,3 +152,37 @@ some interactive options called `Embed elements`_::
...
+                    ->url('https://symfony.com/'),
+            ])
+        );
+
+    // Add the custom options to the chat message and send the message
+    $chatMessage->options($telegramOptions);
+
+    $chatter->send($chatMessage);
+
+.. _`message options`: https://core.telegram.org/bots/api
```
Comment (OskarStark) [inline notifier/chatters.rst]:
```suggestion
.. _`Embed elements`: https://discord.com/developers/docs/resources/webhook
.. _`message options`: https://core.telegram.org/bots/api
```

**Review by OskarStark — CHANGES_REQUESTED** [review]
Please move the link definition of the previous example to the bottom, thanks.

Feel free to ping me afterwards, so I can merge it 👍

**Comment by CyberJack** [comment]
@OskarStark, I updated the documentation and moved all link definitions to the bottom of the document.

**Review by OskarStark — APPROVED** [review]

**Comment by OskarStark** [comment]
Thank you Peter.

**Pull request merged**